org.eclipse.xtext.common.types/src/org/eclipse/xtext/common/types/GenerateCommonTypes.mwe2

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
```

Comment:
I'd prefer if we wouldn't ship this workflow file.

Reply:
Since MWE2 uses common.types, I'd prefer if we could regenerate without an MWE dependency. Otherwise changes to the types model would require MWE to run successfully before other changes to the types model can be applied.